app/oauth.go

```diff
@@ -433,19 +438,19 @@ func (a *App) DeauthorizeOAuthAppForUser(userId, appId string) *model.AppError {
 	}
 
 	// revoke app sessions
```

Comment:
This comment was left untouched though?

Reply:
Thanks for the hint. Missed this one, fixed.